src/btree/bt_vrfy.c

```diff
@@ -699,16 +703,20 @@ __verify_tree(
             ++vs->depth;
             ret = __wt_page_in(session, child_ref, 0);
 
-            /*
-             * If configured, continue traversing through the pages of the tree even after
-             * encountering errors reading in the page.
-             */
-            if (vs->read_corrupt && ret != 0) {
+            if (ret != 0) {
+                if (vs->dump_address)
+                    WT_TRET(__wt_msg(session, "ERROR unreadable row internal page: %s",
+                      __verify_addr_string(session, ref, vs->tmp1)));
+                /*
+                 * If configured, continue traversing through the pages of the tree even after
```

Comment:
We should move this comment to line 716. 

and maybe start like, "If read_corrupt is configured.."

Reply:
fixed https://github.com/wiredtiger/wiredtiger/pull/10095/commits/a9cd31b0c84410b9cc50bb724de16676c2ae1e1d